aha-table.html

```diff
@@ -237,7 +243,7 @@
        * Property to set the the total number of entries in the table,
        * assuming that dataRemote is true and the data provided is only
        * for the visible page.
-       * 
```

Comment:
Let's make sure that these whitespace diffs aren't apparent in the PR to PX team.